tests/e2e/sample-applications/mysql-persistent/pvc-twoVol/cephfs.yml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: List
+items:
+  - apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      name: mysql
+      namespace: mysql-persistent
+      labels:
+        app: mysql
+    spec:
+      accessModes:
+      - ReadWriteOnce
+      storageClassName: cephfs
```

Comment:
I think this should be `ocs-storagecluster-cephfs`

Reply:
the storage class we create as a copy is simply called cephfs for simplicity.  It could be named boogiewoogie but needs to match the storage class we create which in these examples is called simply, cephfs.

Reply:
So the storageClass used for examples would be the default cephfs storageClass and not the one with `backingSnapshot: true`

Reply:
@shubham-pampattiwar +1

Reply:
I don't understand these comments yet.. sorry I am dumb